src/main/java/nl/tudelft/dnainator/ui/models/GraphItem.java

```diff
@@ -6,28 +6,35 @@
 import java.util.stream.Collectors;
 
 import javafx.geometry.Bounds;
+import javafx.scene.Group;
 import javafx.scene.paint.Color;
 import javafx.scene.shape.Rectangle;
 import nl.tudelft.dnainator.core.impl.Cluster;
 import nl.tudelft.dnainator.graph.Graph;
 import nl.tudelft.dnainator.graph.impl.Neo4jSingleton;
+import nl.tudelft.dnainator.ui.drawables.ClusterDrawable;
+import nl.tudelft.dnainator.ui.drawables.DrawableEdge;
 
 /**
- * The {@link GraphItem} class represents the top level object in the viewable model.
- * It is a {@link CompositeItem}, that can hold both content and children.
+ * The {@link GraphItem} class represents the graph that contains the DNA strain.
+ * It holds both content and children, and toggles what to load and display based on the zoom level.
  */
-public class GraphItem extends CompositeItem {
+public class GraphItem extends Group {
+	public static final int RANK_WIDTH = 10;
+	public static final int NO_CLUSTERS = 33000;
 	private static final int FOUR = 4;
```

Comment:
I can see this represents a 4, but what does the 4 mean? Is it a quarter of some specific variable/constant? I would prefer another name for this.